projects/1.16/assets/draconic-evolution/draconicevolution/lang/zh_cn.json

```
@@ -44,240 +48,245 @@
+  "gui.draconicevolution.item_config.toggle_advanced.info": "切换高级配置模式",
+  "gui.draconicevolution.item_config.toggle_global_binding.info": "全局绑定\n允许你在此GUI关闭时使用该按键绑定。\n无论情况如何，不论此按键有没有冲突。",
   "gui.draconicevolution.item_config.toggle_hidden.info": "显示/隐藏物品栏",
-  "gui.draconicevolution.item_config.toggle_preset.info": "Toggle preset mode",
-  "gui.draconicevolution.modular_item.module_grid": "Module Grid",
+  "gui.draconicevolution.item_config.toggle_preset.info": "切换预设模式\n将此属性组转换为属性预设。",
+  "gui.draconicevolution.modular_item.module_grid": "模块网格",
   "gui.draconicevolution.modular_item.modules": "模块",
   "gui.draconicevolution.modular_item.name": "配置模块",
-  "gui.draconicevolution.modular_item.no_module_hosts": "You do not have any modular items in your inventory!",
-  "gui.draconicevolution.modular_item.open_item_config.info": "Open Item Configuration GUI",
+  "gui.draconicevolution.modular_item.no_module_hosts": "你的物品栏中没有任何模块！",
+  "gui.draconicevolution.modular_item.open_item_config.info": "打开物品配置GUI",
   "gui.draconicevolution.reactor.activate": "激活",
   "gui.draconicevolution.reactor.chaos_out": "混沌碎片（输出）",
   "gui.draconicevolution.reactor.charge": "充能",
   "gui.draconicevolution.reactor.convert_rate": "燃料消耗率",
-  "gui.draconicevolution.reactor.convert_rate.info": "This is how fast the reactor is currently using fuel. As the reactor saturation increases, this will go down.",
+  "gui.draconicevolution.reactor.convert_rate.info": "这里会显示反应堆燃料消耗速度。当反应堆稳定后数值会慢慢下降。",
   "gui.draconicevolution.reactor.core_volume": "核心体积",
-  "gui.draconicevolution.reactor.core_volume.info": "This shows the total volume of matter within the reactor in cubic meters (Draconium + Chaos). This value will only change when you add or remove fuel.",
+  "gui.draconicevolution.reactor.core_volume.info": "这里会显示维持当前的控制场强度所需的OP输入功率。这一数值会随着控制场强度呈指数增长。",
   "gui.draconicevolution.reactor.energy_saturation": "能量储量",
   "gui.draconicevolution.reactor.field_rate": "控制场输入功率",
-  "gui.draconicevolution.reactor.field_rate.info": "This is the exact OP/t input required to maintain the current field strength. As field strength increases, this will increase exponentially.",
+  "gui.draconicevolution.reactor.field_rate.info": "这里会显示维持当前的控制场强度所需的OP输入功率。这一数值会随着控制场强度呈指数增长。",
   "gui.draconicevolution.reactor.field_strength": "控制场强度",
   "gui.draconicevolution.reactor.fuel_conversion": "燃料消耗进度",
   "gui.draconicevolution.reactor.fuel_in": "燃料（输入）",
   "gui.draconicevolution.reactor.gen_rate": "产率",
-  "gui.draconicevolution.reactor.gen_rate.info": "This is the current OP/t being generated by the reactor.",
-  "gui.draconicevolution.reactor.go_boom_now": "Emergency shield reserve is now active but it wont last long! There is no way to stop the overload; the stabilizers are fried. I suggest you run!",
+  "gui.draconicevolution.reactor.gen_rate.info": "这里会显示反应堆产生的当前输出功率（OP/t）。",
+  "gui.draconicevolution.reactor.go_boom_now": "应急护盾现在处于激活状态，但不会持续很长时间！无法挽回反应堆过载；反应堆稳定装置已失效。我劝你赶紧跑！",
```

微调
```suggestion
  "gui.draconicevolution.reactor.go_boom_now": "应急护盾已激活，但不能支撑很长时间！无法挽回反应堆过载；反应堆稳定装置已失效。我劝你赶紧跑！",
```
“挽回”那个分句也有点怪，但是没想到好改法

“the stablizers are fried”是否要更口语化一点？

---

```suggestion
  "gui.draconicevolution.reactor.go_boom_now": "应急护盾已激活，但撑不了多久！无法阻止反应堆过载；稳定装置已停摆。我劝你赶紧跑！",
```

---

```suggestion
  "gui.draconicevolution.reactor.go_boom_now": "应急护盾已激活，但它撑不了多久！反应堆过载已成定局；稳定装置彻底失效。为了你的生命安全，快跑！",
```